.github/workflows/ci.yml

```
@@ -16,7 +16,9 @@
       - uses: actions/checkout@v4
 
       - name: Install pnpm
-        run: corepack enable pnpm
+        run: |
+        corepack enable
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 20: could not find expected ':'

(syntax-check)